src/OrchardCore.Modules/OrchardCore.Users/Views/UserActionsMenu.cshtml

```diff
@@ -14,7 +14,7 @@
 @if (canEdit)
 {
     <li>
-        <a asp-action="EditPassword" asp-route-id="@Model.User.UserId" class="dropdown-item">@T["Change password"]</a>
+        <a asp-action="EditPassword" asp-route-id="@Model.User.UserName" class="dropdown-item">@T["Change password"]</a>
```

Comment:
Why is this change needed? I rather see UserId being used.

Reply:
It's for a reason, because the `UserMenu` I'm unable to access the user id directly

Reply:
But it existed before.

Reply:
Nope, there was a bug reported previously, anyhow I used the user id again